internal/pkg/cli/svc_init.go

```diff
@@ -405,6 +397,39 @@ func (o *initSvcOpts) askImage() error {
 	return nil
 }
 
+func (o *initSvcOpts) validExistingSvc() (bool, error) {
```

Comment:
Just brainstorming 💭  What do you think of repurposing this function to read + validate strictly only fields that are in the manifest (i.e. `wkldType`)? 

We can move `validateSvcName` and `validateDuplicateSvc` out of the function. Although there are some repetitive code, it seems to me that it might be clearer what the method is responsible for 🤔 

```suggestion
func (o *initSvcOpts) validateAgainstExistingManifest() (bool, error) {
```

Reply:
That's exactly where I was! Then I felt something was off because it's more like I can't find a proper name of the function. But since this name is still not good enough, I agree probably reverting the change is better.